test-cases/scylla-operator/longevity-scylla-operator-3h-multitenant.yaml

```diff
@@ -26,12 +26,17 @@ k8s_n_loader_pods_per_cluster: 2
 
 k8s_n_auxiliary_nodes: 3
 
-nemesis_class_name: ['FreeTierSetMonkey', 'FreeTierSetMonkey']
+nemesis_class_name: ['K8sSetMonkey', 'K8sSetMonkey']
```

Comment:
maybe it's o.k. to have only one using `K8sSetMonkey`, since those are affecting both, and blocking

Reply:
If to have different one, I would propose to have `FreeTierSetMonkey`, not `Noop` one.

Reply:
Yes, that's what I was aiming for

Reply:
Done